### PR TITLE
URS-580 Implement Cyress tests for Search related pages

### PR DESCRIPTION
**Pull request by darrowcoucla**
Connected to [URS-580](https://jira.library.ucla.edu/browse/URS-580)

#### Acceptance criteria:

- [x] Each of the following is automated via our cypress suite:

    - [x] Search with blank keyword
    - [x] Search Unicorn under all fields
    - [x] Search postcards in titles
    - [x] Search keyword Engineer under subjects field
    - [x] Back to search Results
        - [x] Click the "text" on search results block
            - [x] *This test is done on the Facets ticket*
        - [x] Click on "Armenian Manuscripts" on search results block
        - [x] Click on "Still Image" on search results page
        - [x] Click on Photographer: Artistic Photo on search results block

- [x] A `percy.io` screenshot is taken and submitted at each page
- [x] Linted
- [x] Tested

**Files*

---

+ `e2e/cypress/integration/homepage.js`
+ `e2e/cypress/integration/search.js`
+ `e2e/cypress/support/index.js`
+ `e2e/package-lock.json`
+ `e2e/package.json`

### CODE REVIEW

File: e2e/cypress/integration/search.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/// <reference types="Cypress" />
+beforeEach(() => {
+  cy.visit('https://digital.library.ucla.edu/');
+});
+describe('Search', () => {
```
Comment (andrewbenedictwallace) [inline e2e/cypress/integration/search.js:5]: Can we have blank lines between blocks?

File: e2e/cypress/integration/search.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/// <reference types="Cypress" />
+beforeEach(() => {
+  cy.visit('https://digital.library.ucla.edu/');
```
Comment (andrewbenedictwallace) [inline e2e/cypress/integration/search.js:3]: visit '/', not digital.library.ucla.edu

File: e2e/cypress/support/index.js [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
+import 'cypress-pipe'
```
Comment (andrewbenedictwallace) [inline e2e/cypress/support/index.js:18]: Why is this added? What does it do? Is it necessary?

File: e2e/cypress/integration/search.js (hunk 1 above)
Comment (darrowcoucla) [reply e2e/cypress/integration/search.js:5]: Blank line inserted

File: e2e/cypress/integration/search.js (hunk 2 above)
Comment (darrowcoucla) [reply e2e/cypress/integration/search.js]: Proper target inserted - "/"

File: e2e/cypress/support/index.js (hunk 3 above)
Comment (darrowcoucla) [reply e2e/cypress/support/index.js]: import 'cypress-pipe' removed.

**Comment by darrowcoucla** [comment]
checking test failures

File: e2e/package.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
   "license": "ISC",
   "devDependencies": {
     "@percy/cypress": "^2.3.1",
-    "cypress": "^4.9.0"
+    "cypress": "^4.9.0",
+    "cypress-pipe": "^1.7.0"
```
Comment (andrewbenedictwallace) [inline e2e/package.json:14]: If we don't need cypress-pipe, can we just revert package.json and package-lock.json?

**Comment by darrowcoucla** [comment]
moved to [PR-868](https://github.com/UCLALibrary/ursus/pull/868) after rebase problems
